content/en/docs/languages/go/getting-started.md

```diff
@@ -104,7 +104,7 @@ go get "go.opentelemetry.io/otel" \
   "go.opentelemetry.io/otel/sdk/metric" \
   "go.opentelemetry.io/otel/sdk/resource" \
   "go.opentelemetry.io/otel/sdk/trace" \
-  "go.opentelemetry.io/otel/semconv/v1.21.0" \
+  "go.opentelemetry.io/otel/semconv/v1.24.0" \
```

Comment:
non blocking comment, @open-telemetry/go-approvers: we can implement an automatic version updating similar to what java, collector, etc have:

* https://github.com/open-telemetry/opentelemetry.io/blob/main/.github/workflows/auto-update-versions.yml
* https://github.com/open-telemetry/opentelemetry.io/blob/6e3bf5839719f5e01dda8d90e8cdbc55f1f5c3e0/content/en/docs/languages/java/_index.md?plain=1#L7

Reply:
This could be nice. However, take notice that the version is not equal version of OTel Go. It may not be easy to create such script.

Reply:
Also for java we have different versions for SDK, semconv, etc. So we might be able to replicate that.

Reply:
I also created https://github.com/open-telemetry/opentelemetry-go/pull/4846 so that at least the example would not break in runtime if the code uses an older version of semconv.